### DISCUSSION TITLE
Movies?

### DISCUSSION THREAD

**Discussion by sfrats27** [liamhtml/PLAYHYDRA#8] [Ideas]
Hey, thanks for adding this. I've added your code to my own website hosted off bluehost.com for my school to use, and built on it adding other games like Shellshockers and that momo horror game. I'm quite new to this whole thing, but your project is massively helping me learn html. Atm, as I wait for more game suggestions, I'm looking for a way to bypass youtube/netflix/other streaming sites for movies and embed it into the website. If you're interested in collabing on it or whatever lmk.

btw, entirely new to github n stuff so idk if this is how formatting supposed to go

**Comment by liamhtml** [comment]
hi, it's cool to know that this stupid lil game site could be inspiring *anyone* to learn html! it's cool that you're hosting another version of the site, maybe share the url so everyone can enjoy that site as well. also you should know that once you figure out github, you can always contribute here!

**Comment by sfrats27** [comment]
I've been interested in programming for awhile, late last year I dabbled in C# and made a recoil script for the game Rust. Beginning of the year, I dabbled in other projects for backdoor testing like RATs and now I'm in the website building phase. My website is: https://www.sfrats.website  the front page you'll go on doesn't really go anywhere but I plan to make it a nice organized "link tree" type of place. This project is gonna be found under the link "NEW: Tons of free games!".

**Reply by liamhtml** [reply]
that's cool! i appreciate that you left credit too!

**Reply by sfrats27** [reply]
Of course, I am just dabbling in a little bit of everything until something clicks so I really need to get a project going like getting movies added into the site or something idk
